cli/js/lib.deno.ns.d.ts

```diff
@@ -12,7 +12,7 @@ declare namespace Deno {
    * See: https://no-color.org/ */
   export let noColor: boolean;
 
-  export type TestFunction = () => void | Promise<void>;
+  type TestFunction = () => void | Promise<void>;
```

Comment:
Hmmm, this won't work? There are a few places where `Deno.TestFunction` is used

Reply:
But tests pass...

Reply:
It seems not exporting it causes it to be expanded in doc, so people will see `() => void | Promise<void>` instead of `TestFunction`. That's what I want here as it's a lot more helpful.

Reply:
> It seems not exporting it causes it to be expanded in doc, so people will see `() => void | Promise<void>` instead of `TestFunction`.

Or not... I thought I observed that at some point.